sass/_components.scss

```diff
@@ -0,0 +1,174 @@
+.about {
+  &__container {
+    padding: 0 1.6rem 6rem 1.6rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    position: relative;
+    top: 4rem;
+
+    @media screen and (min-width: 768px) {
+      top: 10rem;
+      max-width: 80rem;
+      margin: 0 auto;
+      justify-content: flex-start;
+    }
+  }
+
+  .hero__welcome {
+    background: #fff;
+    border: 1px solid #e6e6e6;
+  }
+
+  &__contact {
+    font-size: 1.6rem;
+    margin-top: 2rem;
+    font-weight: 400;
+    color: var(--color-secondary-dark);
+  }
+
+  &__email {
+    text-decoration: underline;
+    color: var(--color-primary);
+    font-weight: 700;
+    font-size: 1.4rem;
+  }
+}
+
+.logo {
+  display: flex;
```

Comment:
- [ ] **Optional:** You have used the rule `display: flex;` multiple times, you could extract it into its own class and add it to your `HTML`. It's strongly suggested but not required.